### DISCUSSION TITLE
Question: Multiple payment plan options.

### DISCUSSION THREAD

**Discussion by ohermans1** [kinngh/shopify-node-express-mongodb-app#82] [Q&A]
Hi mate

First of all, thanks again for creating this template! I have found it super handy, and have even managed to get my [first app published](https://apps.shopify.com/autoblogger) :). Of course, some sponsor money will be coming your way if it makes any income!

Just a quick question if you have time, feel free to disregard it, as this isn't an issue with your repo or anything. I'm just wondering if you have any tips on how to setup additional recurring payment options. For example, a $10 per month, $15 per month, and $20 per month. 

I can see from some research that it seems like this isn't handled natively by the billing API, so we need to intercept the routes after OAuth, then let the user select a plan, which then modifies the recurringSubscription request body. I'm just not certain of the best way to do this in your template. 

I would really appreciate any suggestions if you have time. 

Cheers

Ollie

**Comment by kinngh** [comment]
You can create a new `subscriptions` model and store all subscription data there, and on first load check for availability and redirect accordingly. I'd recommend subscribing to the subscriptions/update webhook and updating the db as soon as you have a webhook call.

Also, please know that all questions like this should be a discussions and tagged as `Q&A` instead of being an issue. Putting things in the right place gets a reply faster.
